examples/nanoChatGPT/models/actor_critic.py

```diff
@@ -42,8 +42,13 @@ def __init__(self, base_model):
                 return_log_prob=True,
             ),
         )
-        value_head = TensorDictModule(
-            value_head, in_keys=["x"], out_keys=["state_value"]
+        value_head = TensorDictSequential(
+            TensorDictModule(
+                value_head, in_keys=["x"], out_keys=["state_value"]
+            ),
+            TensorDictModule(
+                lambda x: x[:, -1, :], in_keys=["state_value"], out_keys=["state_value"]
+            ),
```

Comment:
Tell me if I'm wrong but I feel that for the reward, we take the last element, but for the value we don't. Is this expected?